client/controllers/public/channel.js

```diff
@@ -0,0 +1,9 @@
+Template.channel.onCreated(function() {
+  Meteor.subscribe('oneChannel', FlowRouter.getParam('id'));
+});
+
+Template.channel.helpers({
+  channel: function() {
+    return Channel.getChannel(FlowRouter.getParam('id')).fetch()[0];
```

Comment:
If anyone can give me a better way to do this, feel free to put it here. I tried a bunch of different stuff.